elements/lisk-p2p/test/integration/outbound_shuffling.ts

```diff
@@ -61,12 +63,12 @@ describe('Outbound peer shuffling', () => {
 	});
 
 	it('should shuffle outbound peers and close connection with evict', async () => {
-		await wait(500);
+		await wait(1500);
```

Comment:
Why the increased wait here?

Reply:
We overwrite the default wait time(1000ms) in the `createNetwork()` so we need to add it here.